modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -791,9 +813,19 @@ public boolean onUpdate(@NotNull WatchEvent evt) {
 
                             logicalTopology = newLogicalTopology;
 
-                            zonesConfiguration.distributionZones().value().namedListKeys()
+                            Stream.of(
```

Comment:
I'd rather encapsulate all actions to be performed with zone  into some fooBar method and simply call fooBar for defaultZone and fooBar for all other zones from the list.

Reply:
Fixed, thanks.